src/components/trends/smbg/SMBGDayLineAnimated.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+// TODO: this component should render a line connecting the smbgs from a single day
+// it has two or three potential states:
+// - grouping on: the y-position of each point on the line segment
+//   is the average of all the smbgs in the group
+//   NB: when grouping is on, no line hover interaction
+// - grouping off: just connect the dots!
+//   but also include a 2nd, fatter invisible line with onMouseOver & onMouseOut handlers
+//   for the line to "focus" the whole day
+//   (there seems to be a regression on prod re: the rendering of the fatter invisible lines)
+// - [maybe] day is focused (through hover) fatter & solid line connecting the dots
+//   this style also applies when a single smbg is focused
+
+import React, { PropTypes } from 'react';
+import { TransitionMotion, spring } from 'react-motion';
+import { line } from 'd3-shape';
+import _ from 'lodash';
+import cx from 'classnames';
+
+import { THREE_HRS } from '../../../utils/datetime';
+import { findBinForTimeOfDay } from '../../../utils/trends/data';
+
+import styles from './SMBGDayLineAnimated.css';
+
+const SMBGDayLineAnimated = (props) => {
```

Comment:
I'm getting this in the console, which I think we should figure out before this branch gets merged:
<img width="642" alt="screenshot 2016-10-25 11 08 27" src="https://cloud.githubusercontent.com/assets/1588547/19700895/fae4f6b2-9aad-11e6-8fbe-f562ab2c735c.png">

LMK if you need help with the detective work @jh-bate